app/views/layouts/application.html.erb

```diff
@@ -11,6 +11,17 @@
   </head>
 
   <body>
+    <div class='notice_alert'>
+      <p class="notice"><%= notice %></p>
+      <p class="alert"><%= alert %></p>
+    </div>
+
+    <div class='link_to_sign'>
+      <% if user_signed_in? %>
+        Logged in as <strong><%= current_user.email %></strong>.
+        <%= link_to "Logout", destroy_user_session_path, method: :delete, :class => 'navbar-link'  %>
```

Comment:
- [ ] How about displaying the current_user name instead of their email, what do you think?
![Screen Shot 2021-12-24 at 7 16 53 AM](https://user-images.githubusercontent.com/50230297/147327481-df6a434f-bdc3-4cdc-b130-04f2b4f02705.png)
- [ ] Kindly refer to the original design by [Gregoire Vella](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=) and try improve the design and fonts of the application, margins and font-size of text

Reply:
Good idea, I'll change it to the current user name